vips/vip-408/bscmainnet.ts

```diff
@@ -0,0 +1,182 @@
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [ethereum.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [ethereum.POOL_REGISTRY, parseUnits("10000", 18)],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: VsUSDe_Ethena,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VsUSDe_Ethena,
+            parseUnits("0.9", 18),
+            parseUnits("0.92", 18),
+            parseUnits("10000", 18),
+            VTOKEN_RECEIVER,
+            parseUnits("50000000", 18),
+            parseUnits("0", 18),
+          ],
+        ],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [USDC, parseUnits("10000", 6), ethereum.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: USDC,
+        signature: "approve(address,uint256)",
+        params: [ethereum.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: USDC,
+        signature: "approve(address,uint256)",
+        params: [ethereum.POOL_REGISTRY, parseUnits("10000", 6)],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: VUSDC_Ethena,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VUSDC_Ethena,
+            parseUnits("0", 18),
+            parseUnits("0", 18),
+            parseUnits("10000", 6),
+            VTOKEN_RECEIVER,
+            parseUnits("50000000", 6),
+            parseUnits("46000000", 6),
+          ],
+        ],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: VsUSDe_Ethena,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.010", 18)],
+        dstChainId: LzChainId.ethereum,
+      },
+
+      {
+        target: VUSDC_Ethena,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.020", 18)],
+        dstChainId: LzChainId.ethereum,
+      },
+
+      {
+        target: COMPTROLLER_ETHENA,
+        signature: "setActionsPaused(address[],uint8[],bool)",
+        params: [[VsUSDe_Ethena, VUSDC_Ethena], [2, 7], true],
```

Comment:
We cannot pause the "enter market" action on the USDC market, because that market is borrowable, and the `Comptroller.preBorrowHook` adds the borrowable market to the list of user markets (and to do that, it checks the permission on the action ENTER_MARKET, see it [here](https://github.com/VenusProtocol/isolated-pools/blob/develop/contracts/Comptroller.sol#L1509))

Reply:
[1866d37](https://github.com/VenusProtocol/vips/pull/440/commits/1866d377e77845c38ab5502842a9fee5915bfbb9)